### PR TITLE
Check glide version

### LINKED ISSUE #18: Glide 0.13 minimum is required
To be able to get proper results out of make update-deps a version of glide no lower than 0.13.0 is needed.
This should probably be enforce by the make target via a version check.
Otherwise the tools pulls in garbage.

### PR DESCRIPTION
**Pull request by soltysh**
Fixes #18.

@deads2k ptal

### CODE REVIEW

File: hack/update-deps.sh [hunk 1]
```diff
@@ -1,4 +1,23 @@
 #!/bin/bash -e
 
+readonly REQUIRED_GLIDE_VERSION="v0.13"
+
+function verify_glide_version() {
+    if ! command -v glide >/dev/null 2>&1; then
```
Comment (sttts) [inline hack/update-deps.sh]: `&> /dev/null`

Comment (sttts) [reply hack/update-deps.sh]: indentation is off

File: hack/update-deps.sh [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+		echo "[FATAL] Glide was not found in \$PATH. Please install version ${REQUIRED_GLIDE_VERSION} or newer."
+        exit 1
+	fi
+
+	local glide_version
+	glide_version=($(glide --version))
+	if [[ "${glide_version[2]}" < "${REQUIRED_GLIDE_VERSION}" ]]; then
```
Comment (sttts) [inline hack/update-deps.sh]: this is string comparison? Dot is 0x2e, i.e. smaller than digits. Hence, I believe this line is correct, even with glide 1.9 and older.

**Comment by sttts** [comment]
Some nits. Otherwise lgtm.

File: hack/update-deps.sh (hunk 2 above)
Comment (soltysh) [reply hack/update-deps.sh]: This will work with 0.9.0, just checked, so it's fine to have it as is.

**Comment by soltysh** [comment]
Nits updated.

**Pull request merged**